exercises/hangman/hangman_test.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+
+from example import Hangman
+
+
+# Adapt from c-sharp track
```

Comment:
Please change to the following:
`# Tests adapted from csharp//hangman/HangmanTest.cs`

Normally, we would have a comment here citing the version of the canonical data that these tests were based on. In this case, there is no canonical data for this exercise, so this is fine.

Reply:
@khoa102 Please apply this fix.